cdk/kraken/package.json

```diff
@@ -6,55 +6,53 @@
     "url": "https://github.com/pennlabs/infrastructure.git",
     "directory": "cdk/kraken"
   },
-  "bin": {},
   "scripts": {
-    "projen": "npx projen",
-    "start": "npx projen start",
+    "build": "npx projen build",
     "compile": "npx projen compile",
+    "default": "npx projen default",
+    "docgen": "npx projen docgen",
+    "eject": "npx projen eject",
+    "eslint": "npx projen eslint",
+    "package": "npx projen package",
+    "post-compile": "npx projen post-compile",
+    "pre-compile": "npx projen pre-compile",
     "test": "npx projen test",
-    "build": "npx projen build",
-    "bump": "npx projen bump",
-    "release": "npx projen release",
-    "test:watch": "npx projen test:watch",
     "test:update": "npx projen test:update",
-    "projen:upgrade": "npx projen projen:upgrade",
+    "test:watch": "npx projen test:watch",
     "watch": "npx projen watch",
-    "package": "npx projen package",
-    "eslint": "npx projen eslint",
-    "docgen": "npx projen docgen"
+    "projen": "npx projen"
   },
   "author": {
     "name": "Penn Labs",
     "url": "https://pennlabs.org",
     "organization": "Penn Labs"
   },
-  "homepage": "https://kraken.pennlabs.org",
   "devDependencies": {
-    "@types/jest": "^26.0.19",
-    "@types/node": "^10.17.0",
-    "@typescript-eslint/eslint-plugin": "^4.3.0",
-    "@typescript-eslint/parser": "^4.3.0",
-    "codecov": "^3.8.1",
-    "eslint": "^7.16.0",
-    "eslint-import-resolver-node": "^0.3.4",
-    "eslint-import-resolver-typescript": "^2.3.0",
-    "eslint-plugin-import": "^2.22.1",
-    "jest": "^26.6.3",
-    "jest-junit": "^12",
-    "json-schema": "^0.2.5",
-    "projen": "^0.8.0",
-    "standard-version": "^9.0.0",
-    "ts-jest": "^26.4.4",
-    "typedoc": "^0.19.2",
-    "typescript": "^3.9.5"
+    "@types/jest": "^27.4.1",
+    "@types/node": "^12",
+    "@typescript-eslint/eslint-plugin": "^5",
+    "@typescript-eslint/parser": "^5",
+    "codecov": "^3.8.3",
+    "eslint": "^8",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-node": "^0.3.6",
+    "eslint-import-resolver-typescript": "^2.7.1",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-prettier": "^4.0.0",
+    "jest": "^27.5.1",
+    "jest-junit": "^13",
+    "json-schema": "^0.4.0",
+    "prettier": "^2.6.2",
+    "projen": "^0.54.7",
+    "ts-jest": "^27.1.4",
+    "typedoc": "^0.21.4",
+    "typescript": "^4.6.3"
   },
-  "peerDependencies": {},
   "dependencies": {
-    "cdkactions": "^0.2.0",
-    "constructs": "^3.2.80",
+    "cdkactions": "^0.2.3",
+    "constructs": "^10.0.106",
```

Comment:
Why is constructs `^10.0.106` here and `^3.3.259` in the overall `cdk` folder?

Reply:
Oh well

Reply:
Ran `yarn tsc` and the new kittyhawk lib on https://github.com/joyliu-q/kittyhawk-demos and it worked as intended (no changes)